dluhc-component-library/src/components/maps/components/BaseMap.tsx

```diff
@@ -4,8 +4,8 @@ import { useGeographic } from "ol/proj";
 import { OSM } from "ol/source";
 import { CSSProperties } from "preact/compat";
 import { useEffect, useMemo, useRef } from "preact/hooks";
-import "../../../node_modules/ol/ol.css";
-import { useMap } from "../../contexts/mapContext";
+import { useMap } from "src/contexts/mapContext";
+import "../../../../node_modules/ol/ol.css";
```

Comment:
`Non-blocker` This doesn't feel like the way we should be doing this, can we have a chat about it at some point please?